Packs/SplunkPy/Integrations/SplunkPy/SplunkPy_test.py

```diff
@@ -1122,6 +1122,53 @@ def test_build_search_human_readable(mocker):
     assert headers == expected_headers
 
 
+def test_build_search_kwargs():
+    args = {'earliest_time': '2021-11-23T10:10:10', 'latest_time': '2021-11-23T10:10:20', 'app': 'test_app', 'polling': False}
```

Comment:
please add given-when-then (also in the 2nd test)

Reply:
done.